tests/python-pytest/onnx/onnx_backend_test.py

```diff
@@ -57,37 +57,40 @@
     'test_floor',
 
     ## Joining and spliting
-    #'test_concat.*',  #---Failing test
+    'test_concat',
 
     #Basic neural network functions
     'test_sigmoid',
     'test_relu',
-    #'test_constant_pad',
-    #'test_edge_pad',
-    #'test_reflect_pad',
+    'test_constant_pad',
+    'test_edge_pad',
+    'test_reflect_pad',
     'test_matmul',
     'test_leakyrelu',
     'test_elu',
-    #'test_softmax*',
+    'test_softmax_example',
+    'test_softmax_large_number',
+    'test_softmax_axis_2',
     'test_conv',
     'test_basic_conv',
-    #'test_globalmaxpool',
-    #'test_globalaveragepool',
-    #'test_batch_norm',
+    'test_transpose',
+    #'test_globalmaxpool', - tests to be added
+    #'test_globalaveragepool', - tests to be added
+    #'test_batch_norm', - tests to be added
+    #'test_gather',
```

Comment:
Can you please add these tests as part of this PR, because, these operators are added in this PR.

Reply:
these tests are to be added in the ONNX repo here - https://github.com/onnx/onnx/tree/master/onnx/backend/test/case/node. The ONNX team is still working on them. I will remove these comments for now. One ONNX updates these operators and their tests, we will update our code.

Reply:
Does this mean, for an operator we are adding, we do not cover with test cases? May be prone for issues?

Reply:
So ONNX adds two type of tests , one that @anirudhacharya  pointed, another is they create models with just single nodes https://github.com/onnx/onnx/tree/master/onnx/backend/test/data/model/pytorch-converted

So the plan is , if we do not have tests at either of the places we create a PR in onnx test repo for the test. In this case we have Batch norm test in https://github.com/onnx/onnx/tree/master/onnx/backend/test/data/model/pytorch-converted

batch_norm comes under above category.
'Gather': We can actually remove this Gather operation, since its only partially supported, and can't enable the tests. 

'Globalaveragepool'/'Globalmaxpool' will add them.

Reply:
Makes sense.